css/footer.css

```diff
@@ -0,0 +1,140 @@
+footer {
+  width: 100%;
+  background: var(--black);
+  margin-top: 2em;
+  font-size: 5vw;
+  font-weight: bolder;
+  color: var(--white);
+}
+.footer h1 {
+  letter-spacing: 0.3em;
+  font-weight: 100;
+}
+.footer_info {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+.footer_item {
+  padding-top: 3em;
+}
+.footer_item ul {
+  padding-top: 1em;
+}
+.footer_item ul li {
+  list-style: none;
+  margin-top: 5px;
+}
+.footer_item ul li a {
+  color: white;
+}
+.footer_item h1 {
+  font-size: 8vw;
+}
```

Comment:
The font should be different here. 
Also, I wouldn't use an `h1` here - keep `h1` for the main large title on the page, and use `h2`-`h6` for the rest. A good way to decide which heading tag to use is to make a sort of hierarchy of all the headings in a design, and order their styles by 'importance' (size, thickness etc). Perhaps this is an h4 for example, if compared to the other styles on the page.